README.md

```diff
@@ -252,6 +252,8 @@ After the instructions for your OS (see below), you can use this commands to run
 dub run dpp -- yoursourcefilenamehere.dpp
 ```
 
+Note: to reproduce the environment of dpp's CI, you can run [setup-cpp](https://github.com/aminya/setup-cpp) with `--llvm=11.0.0` and `--vcvarsall=true`. This will set up LLVM 11.0.0 and the proper environment variables.
```

Comment:
Shouldn't this be in the Windows section?

Reply:
setup-cpp is cross platform.